tests/Hosting.UnitTests/HostBuilderExtensionsTests.cs

```diff
@@ -21,6 +22,7 @@ public class HostBuilderExtensionsTests
 		public void AddOmexServices_TypesRegistered(Type type)
 		{
 			object collectionObj = new ServiceCollection()
+				.AddSingleton<IConfiguration>(new ConfigurationBuilder().Build())
```

Comment:
As specified before, consider using the Option pattern. You're also defining it as a singleton, so you can use the `IOptions` interface, but be aware that using this dependency configuration the code will not be able to parse the updated configuration.

Reply:
Is this line still necessary after having implemented the Options pattern?